packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -116,7 +116,13 @@ const ActionInfo = ({
   const errorMessage = (typeof error === 'object' && error?.message) || (typeof error === 'string' && error)
   return (
     <Stack direction="row" columnGap={Spacing.sm} data-testid={testId} sx={sx}>
-      <Typography flexGrow={1} variant={labelSize[size]} color={labelColor ?? 'textSecondary'} textAlign="start">
+      <Typography
+        flexGrow={1}
+        variant={labelSize[size]}
+        color={labelColor ?? 'textSecondary'}
+        textAlign="start"
+        component="div"
```

Comment:
What dom validation error was causing the lack of this specifically? Still feels weird this needs to be turned from a span into a div 🤔

Reply:
The address component in the market pages shows a token icon in the title